src/__tests__/component.test.js

```diff
@@ -1,21 +1,19 @@
 import GLOBALS from '../consts/GLOBALS';
 import component from '../component';
 import Component from '../component/Component';
-import r from '../r';
+import r from '../r'; // eslint-disable-line
```

Comment:
Hmm, just a question, why is this necessary?
Is it because of the short path?

Reply:
r was being imported (I think for the jsx transforms?), but never used. 

I guess we can check out if there's a rule in there for a situation like that

Reply:
Would be convenient